README.md

```diff
@@ -88,52 +88,110 @@ curl -L https://aka.ms/install-dmr-client-linux | bash
 iwr https://aka.ms/install-dmr-client-windows -UseBasicParsing | iex
 ```
 
+#### Update / Uninstall
+
+To update the client you must uninstall the current version first:
+
+```
+dotnet tool uninstall -g microsoft.iot.modelsrepository.commandline
```

Comment:
Good idea to show how to uninstall. Should we include guidance to uninstall beta.1? IIRC the package Id was `dmr-client`

Reply:
added a note